### ISSUE TITLE
Announcements for Mon Dec 5 - Sun Dec 11

### ISSUE THREAD

**Issue by hectorsector** [cop1000/cop1000#167]
👋 @cop1000/all-students you're so close to the end of the semester! Congratulations! 

This week, please continue working on your final project scoping and start implementation after you get the 👍 from me. There's some fantastic conversations going on in the [final project repo](https://github.com/cop1000/final-project). Please note that although proposals were due yesterday, you definitely need to get approval before starting implementation. If you implement an idea that was not approved, you will not be able to get a high score on the final project. 

**Your final project implementation is due no later than Wed, Dec 14 at 11:59pm.**

If you've got any questions at all, I am available as always via Gitter chat, email, or just at-mention me on an open issue or PR.

### ‼️ Reminder

If you haven't already done so, please remember that course evaluations are due tomorrow, Wed, Dec 7. You can fill out your course evaluation by going to http://tiny.cc/CoursEval_Student -- you'll see a listing of all the courses for which you're registered. These evaluations are anonymous and not released to me until the semester is finished, so please provide honest and thorough feedback on the course. I hope this has been a good experience for you ❤️

**Comment by hectorsector** [comment]
Next announcement is at #168
